apps/public-docsite-v9/src/shims/ThemeShim/v9ThemeShim.ts

```diff
@@ -53,7 +53,7 @@ const mapAliasColors = (palette: IPalette, inverted: boolean): ColorTokens => {
     colorNeutralForegroundInvertedPressed: palette.white,
     colorNeutralForegroundInvertedSelected: palette.white,
     colorNeutralForegroundOnBrand: palette.white,
-    colorNeutralForegroundInvertedStatic: palette.white,
+    colorNeutralForegroundStaticInverted: palette.white,
```

Comment:
The reason I went with "inverted static" is because I felt it better aligned with our other usages of "inverted" which are all "inverted {modifier}". To me "static" feels like a modifier much like "pressed" or "hover".

I'm fine with changing it too :)

Reply:
Yeah, that makes sense. But the reason why it is **StaticInverted** is to point out that for this particular token, **Static** it is not a modifier of **Inverted**.